src/Components/Admins/Modals/ModalDeleteConfirmation/ModalDeleteConfirmation.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import styles from './modalDeleteConf.module.css';
+
+function ModalDeleteConfirmation({ show, setShowConfirmationModal, adminFullName }) {
+  if (!show) {
+    return null;
+  }
+
+  const cerrar = () => {
```

Comment:
Since all the code is in english, this should be too. Besides, we could give it a more descriptive name like closeModal